src/data_collector.py

```diff
@@ -0,0 +1,52 @@
+import aiohttp
+import asyncio
+import boto3
+import json
+
+STATUS_NODES = [
+    'node.explorer.hathor.network',
+    'node1.mainnet.hathor.network',
+    'node2.mainnet.hathor.network'
+]
+
+def process_node_status(node_status: dict):
+    session = boto3.session.Session()
+    lamb = session.client("lambda", endpoint_url="http://localhost:3002")
+    lamb.invoke(
+        InvocationType='Event',
+        FunctionName='hathor-explorer-service-dev-network_data_aggregator',
+        Payload=json.dumps(node_status)
+    )
+
+async def request_node(session: aiohttp.ClientSession, url: str):
+    try:
+        async with session.get(url) as resp:
+            status = await resp.json()
+            process_node_status(status)
+    except:
+        pass
+
+async def get_nodes_statuses():
+    async with aiohttp.ClientSession() as session:
+        responses = []
+        for node in STATUS_NODES:
+            responses.append(asyncio.ensure_future(request_node(session, f"https://{node}/v1a/status")))
```

Comment:
What if we have a timeout or any error in the request?

Maybe we could handle the request first and, if succeeds we append to the response, what do you think?

Reply:
This could be done with `aiohttp.ClientSession(timeout=...)` and [`asyncio.gather`](https://docs.python.org/3/library/asyncio-task.html#running-tasks-concurrently) the executions (`asyncio.gather` awaits all executions concurrently). If a request times out, `asyncio.TimeoutError` will be raised by the `get`, so the function could return a noop or a default message, like "node unreachable".

Reply:
I agree that the polling can be smarter. But for now i've decided to keep it simple: do the request and send the response forward, if it succeed. Otherwise, we just don't care. It will be called again next second.